users/style.css

```diff
@@ -27,6 +27,27 @@
   color: var(--white-color);
 }
 
+html,
```

Comment:
@heyrandhir if you can share the screenshots before and after it would be really helpful.

Reply:
sure I will add it

Reply:
Done @vikhyat187